src/components/sequencing/Sequences.svelte

```diff
@@ -59,6 +61,34 @@
     const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
     goto(`${base}/sequencing/new${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}`);
   }
+
+  async function importLibary(): Promise<void> {
+    const library = await effects.importLibrarySequences(workspaceId);
+    if (!library) {
+      return;
+    }
+    let contents = '';
+    try {
```

Comment:
Should this try/catch prevent the logic from proceeding if it can't read the file? The `importLibrarySequences` effect might be better equipped to handle this logic and also return the contents. Then it can show a failure modal as well.

Reply:
I moved things into the effects.importLibrarySequences so everything is handled there. Thank you for the suggestions